app/Phragile/TransactionLoader.php

```diff
@@ -1,6 +1,7 @@
 <?php
 namespace Phragile;
 
+// TODO: rename to e.g. TransactionDataFetcher?
```

Comment:
hmm, but shouldn't it actually be `TransactionRawDataFetcher` as it deals with Phabricator's data (per `TransactionRawDataProcessor`)?